plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -229,6 +229,13 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("create").Groups(authorizationGroup).Resources("subjectaccessreviews").RuleOrDie(),
 			},
 		},
+		{
+			// a role to use for the kube-dns pod
+			ObjectMeta: api.ObjectMeta{Name: "system:kube-dns"},
+			Rules: []rbac.PolicyRule{
+				rbac.NewRule("list", "watch").Groups(legacyGroup).Resources("endpoints", "services").RuleOrDie(),
```

Comment:
> If we get the optional config maps in 1.6, we won't need that permission, right?

The configmap permission could be done as a secondary, optional role attached to your service account.  A cluster-admin could add it if he needed the feature.  We've done that with a few permissions in openshift.  There's a base role that works 90% of the time and if you need more power, you grant more power with a second role: opt-in.